Portal/src/Datahub.Portal/i18n/localization.json

```diff
@@ -1,1272 +1,1540 @@
+  "Azure Databricks is a fast, easy, and collaborative Apache Spark-based analytics platform. Accelerate big data analytics and artificial intelligence (AI) solutions with Azure Databricks, a fast, easy and collaborative Apache Spark-based analytics service.": "Azure Databricks is a fast, easy, and collaborative Apache Spark-based analytics platform. Accelerate big data analytics and artificial intelligence (AI) solutions with Azure Databricks, a fast, easy and collaborative Apache Spark-based analytics service.",
+  "Azure Databricks": "Azure Databricks",
+  "Azure PostgreSQL Database": "Azure PostgreSQL Database",
+  "Azure Postgres": "Azure Postgres",
+  "Azure SQL Database is a scalable database service built for the cloud to store structured datasets and access them through Tableau, Power BI, Web forms or other tools.": "Azure SQL Database is a scalable database service built for the cloud to store structured datasets and access them through Tableau, Power BI, Web forms or other tools.",
+  "Azure Storage Blob": "Azure Storage Blob",
+  "Azure": "Azure",
+  "Azure\u0027s Linux App Service allows you to quickly build, deploy, and scale web apps and APIs developed with .NET, .NET Core, Node.js, Java, Python, or PHP in containers.": "Azure\u0027s Linux App Service allows you to quickly build, deploy, and scale web apps and APIs developed with .NET, .NET Core, Node.js, Java, Python, or PHP in containers.",
+  "Back to DataHub": "Back to DataHub",
+  "Baseline_DESC": "Baseline ",
+  "Beam us up, Scotty": "Beam us up, Scotty",
+  "Below you will find a comprehensive list of the most popular resources on the Datahub. If you are looking for something specific, you can also browse the full list of resources on the navigation sidebar.": "Below you will find a comprehensive list of the most popular resources on the Datahub. If you are looking for something specific, you can also browse the full list of resources on the navigation sidebar.",
+  "Body": "Body",
+  "Branch": "Branch",
+  "Branch_Budget_NUM": "Branch Budget",
+  "Branch_Name": "Branch",
+  "Branch_Optional_DESC": "Branch OR Division",
+  "Branch_TXT": "Branch",
+  "Breadcrumb": "Breadcrumb",
+  "Browse by Sector": "Browse by Sector",
+  "Browse by Type": "Browse by Type",
+  "Bucket Name": "Bucket Name",
+  "Budget": "Budget",
+  "Budget_NUM": "Budget",
+  "Business_Owner": "Business Owner",
+  "Business_Owner_Approval": "The Business Owner (identified above) has endorsed this request.",
+  "C'est mieux comme ça": "C'est mieux comme ça",
+  "CFS": "CFS",
+  "CIOSB": "CIOSB",
+  "CMS - Others": "CMS - Others",
+  "CMSS": "CMSS",
+  "COMMON.No": "COMMON.No",
```

Comment:
Hierarchical translations like this seem to have all been collapsed and replaced with their keys instead of the values they had. Not sure if any of the existing ones are still in use anywhere in the app, but for the Open Data stuff I'm working on I did use the hierarchy. E.g. TbsOpenPublishing.Status.[each status].Title and .Description. I can manually update those ones in the json file for the work I'm doing now, but if we have e.g. COMMON.Yes or ERROR-PAGE.whatever, we would need to dig through the file history to restore the translated values.

Reply:
Yes, please switch to not using the hierarchy.

I believe most of the hierarchy formatted keys were used on legacy components, however any remaining will be obvious enough to fix quickly.